File: app/src/main/java/com/websarva/wings/android/kaigonote/AcsidentoActivity.java
package com.websarva.wings.android.kaigonote;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.view.View.OnClickListener;

import androidx.appcompat.app.AppCompatActivity;

import com.websarva.wings.android.kaigonote.databinding.AcsidentoBinding;


public class AcsidentoActivity extends AppCompatActivity implements OnClickListener {
    private AcsidentoBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = AcsidentoBinding.inflate(getLayoutInflater());
        View view = binding.getRoot();
        setContentView(view);
        binding.btClick22.setOnClickListener(this);//ヒヤリハットボタンがタップされたときの処理
        binding.btClick23.setOnClickListener(this);//事故報告ボタンがタップされたときの処理
    }

    public void onClick(View v) {
        if (v.getId() == R.id.bt_click22) {
            Intent intent22 = new Intent(this, HiyariActivity.class);//ヒヤリハット
            startActivity(intent22);
        } else if (v.getId() == R.id.bt_click23) {
            Intent intent23 = new Intent(this, ZikoActivity.class);//事故報告
            startActivity(intent23);
        }
    }
}

File: app/src/main/java/com/websarva/wings/android/kaigonote/AcsidentoActivity.kt
package com.websarva.wings.android.kaigonote

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.websarva.wings.android.kaigonote.R
import android.content.Intent
import android.view.View
import com.websarva.wings.android.kaigonote.HiyariActivity
import com.websarva.wings.android.kaigonote.ZikoActivity
import com.websarva.wings.android.kaigonote.databinding.AcsidentoBinding

class AcsidentoActivity : AppCompatActivity(), View.OnClickListener {
    private var binding: AcsidentoBinding? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = AcsidentoBinding.inflate(layoutInflater)
        val view: View = binding!!.root
        setContentView(view)
        binding!!.hiyariClick.setOnClickListener(this) //ヒヤリハットボタンがタップされたときの処理
        binding!!.zikoClick.setOnClickListener(this) //事故報告ボタンがタップされたときの処理
    }

    override fun onClick(v: View) {
        if (v.id == R.id.hiyari_click) {
            val intent22 = Intent(this, HiyariActivity::class.java) //ヒヤリハット
            startActivity(intent22)
        } else if (v.id == R.id.ziko_click) {
            val intent23 = Intent(this, ZikoActivity::class.java) //事故報告
            startActivity(intent23)
        }
    }
}